### ISSUE TITLE
sentry-cli logging levels dict significantly changed

### ISSUE THREAD

**Issue by tim-lucida** [getsentry/sentry-cli#739]
hi!

Couple days ago our ci-cd pipelines with sentry-cli usage were broken with the following error

```
error: 'INFO' isn't a valid value for '--log-level <LOG_LEVEL>'
	[possible values: debug, error, info, trace, warn]
USAGE:
    sentry-cli releases --log-level <LOG_LEVEL>
For more information try --help
CREATING RELEASE! 0
error: 'INFO' isn't a valid value for '--log-level <LOG_LEVEL>'
	[possible values: debug, error, info, trace, warn]
USAGE:
    sentry-cli releases --log-level <LOG_LEVEL>
For more information try --help

```
supposedly, logging-levels were broken in https://github.com/getsentry/sentry-cli/commit/d955118fe63bd07a2e6c543d0f998f10d05c94c1

**steps to reproduce**
run the following command
`sentry-cli releases -p my-app-name --log-level=INFO info 61def5a8375371d2221b6cde396aa62e8239032c`

**Comment by tim-lucida** [comment]
Will sentry-cli use capitalised (INFO, WARN, etc) logging-levels in future or should we switch to lowercase ones (info, warn, etc)?

**Comment by kamilogorek** [comment]
Hey! It was my oversight, excuse me. Will be fixed in #740

**Comment by tim-lucida** [comment]
thanks for quick response!
@kamilogorek could you suggest when PR will be merged and hotifx released?

**Comment by kamilogorek** [comment]
@tim-lucida Today afternoon/Tomorrow morning. Have to merge one more feature first.

**Comment by kamilogorek** [comment]
@tim-lucida released as `1.53.0`
